src/components/Cluster/ClusterDetail/__tests__/ScaleClusterModal.js

```diff
@@ -0,0 +1,620 @@
+import '@testing-library/jest-dom/extend-expect';
+
+import { fireEvent, render, wait } from '@testing-library/react';
+import nock from 'nock';
+import React from 'react';
+import { Providers } from 'shared';
+import { StatusCodes } from 'shared/constants';
+import {
+  API_ENDPOINT,
+  AWSInfoResponse,
+  v4AWSClusterResponse,
+  v4AzureClusterResponse,
+  v4KVMClusterResponse,
+} from 'testUtils/mockHttpCalls';
+import { getComponentWithStore } from 'testUtils/renderUtils';
```

Comment:
I think this is something I wanted to avoid for 'component' tests. I'd like to eventually have a clear separation between components that need the store and wrappers that wire the store up to the components. For now, to get this tested , I suppose its ok. It even serves as a signal. But it's something to think about.

Reply:
I agree with that 100%. It would make testing so easy 👍